### PR TITLE
Do not show statistics DataKind in statistics summary if value is NaN

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: src/backend/SyncBackendConnection.cpp [hunk 1]
```diff
@@ -565,13 +565,7 @@ EntityInfo SyncBackendConnection::get_summary(
             configuration.second);                  // StatisticKind
 
         // Check that there is data to report, if not set or NaN give Nan value
```
Comment (jparisu) [inline src/backend/SyncBackendConnection.cpp:567]:
```suggestion
        // Check that there is data to report, if not set or NaN do not show this statistic data.
```

**Review by jparisu — CHANGES_REQUESTED** [review]
NIT

File: src/backend/SyncBackendConnection.cpp (hunk 1 above)
Comment (rsanchez15) [reply src/backend/SyncBackendConnection.cpp]: Addressed in d11fcbc

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**